closed/autoconf/custom-hook.m4

```
@@ -445,6 +446,34 @@ AC_DEFUN([OPENJ9_CONFIGURE_JITSERVER],
   AC_SUBST(OPENJ9_ENABLE_JITSERVER)
 ])
 
+AC_DEFUN([OPENJ9_CONFIGURE_JFR],
+[
+  AC_ARG_ENABLE([jfr], [AS_HELP_STRING([--enable-jfr], [enable JFR support @<:@disabled@:>@])])
```

The default is misleading - by default it _**is**_ enabled on xa64.